src/utils/index.js

```diff
@@ -99,6 +106,12 @@ module.exports.throttleComponentChanged = function (functionToThrottle, minimumI
   };
 };
 
+/**
+ * Identical to throttleLeadingAndTrailing.
+ * Exists for back-compatibility with 1.3.0.
+ */
+module.exports.throttleComponentChanged = module.exports.throttleLeadingAndTrailing;
```

Comment:
I think we can just rename. Don't think many were relying on this. We can add a deprecation note in next release notes.